comit/src/order.rs

```diff
@@ -87,7 +87,7 @@ impl BtcDaiOrder {
 }
 
 /// A newtype representing a quantity in a certain base currency B.
-#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
Where is the `Hash` trait used?

Reply:
It is used for computing the key of the LruCache because the LruCache is basically a bounded hashmap.